pages/fullscreen/playback/index.vue

```diff
@@ -14,13 +14,7 @@
         class="d-flex justify-center"
       >
         <div class="album-cover">
-          <blurhash-image :item="item" @error="onImageError">
-            <template #placeholder>
-              <v-avatar tile size="65vh" color="primary">
-                <v-icon dark x-large style="font-size: 60vh">mdi-album</v-icon>
-              </v-avatar>
-            </template>
-          </blurhash-image>
+          <blurhash-image :item="item" @error="onImageError" />
```

Comment:
Why do we keep the error here? Genuine question

Reply:
Because in the methods, at the end of the file, we clear the backdrop when there's no image, so the backdrop of the previous item that had a picture doesn't persist.